src/components/EnterpriseApp/EnterpriseAppRoutes.jsx

```diff
@@ -84,6 +85,14 @@ const EnterpriseAppRoutes = ({
         />
       )}
 
+      {enableAnalyticsPage && enterpriseAppPage === ROUTE_NAMES.analyticsv2 && (
```

Comment:
Where have we defined `enableAnalyticsPage` ?

Reply:
It is coming from https://github.com/openedx/frontend-app-admin-portal/blob/0639733b83b2a8e77b845162b8aca455e64bad18/src/components/EnterpriseApp/index.jsx#L160